src/parser/core/modules/Buffs.ts

```diff
@@ -47,30 +68,25 @@ class Buffs extends Module {
     ];
   }
 
-  activeBuffs = [];
-  constructor(options) {
-    super(options);
-    this.loadBuffs(this.buffs());
-  }
-  loadBuffs(buffs) {
+  loadBuffs(buffs: SpellbookBuff[]) {
     this.activeBuffs = buffs
-      .map((options) => new this.constructor.BUFF_CLASS(options))
+      .map((options) => new Buffs.BUFF_CLASS(options))
```

Comment:
since this is referencing a static member of this class, it can just be replaced with:
```suggestion
      .map((options) => new Buff(options))
```